Foundation/src/File_UNIX.cpp

```diff
@@ -340,8 +340,12 @@ void FileImpl::copyToImpl(const std::string& path) const
 		handleLastErrorImpl(_path);
 	}
 	const long blockSize = st.st_blksize;
-
-	int dd = open(path.c_str(), O_CREAT | O_TRUNC | O_WRONLY, st.st_mode);
+	int dd;
+	if (options == 1) {
```

Comment:
better check would be `options & File::OPT_FAIL_ON_OVERWRITE` (in case we add another option)

Reply:
So i have to include the `File.h` Header inside the `File_UNIX.cpp`?